From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Thu, 12 Jul 2012 12:06:08 +0100
Subject: use CPPFLAGS

Need CPPFLAGS during compilation to call hardened routines during
hardened build.  See lintian tag hardening-no-fortify-functions.
---
 Makefile |    1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 8e6d793..491fad7 100644
--- a/Makefile
+++ b/Makefile
@@ -57,6 +57,7 @@ OSDEF=	-Dunix
 #				# (requires "s9-real.scm" EXTRA_SCM, above)
 
 DEFS=	$(OSDEF) \
+	$(CPPFLAGS) \
 	-DDEFAULT_LIBRARY_PATH="\".:~/s9fes:$(LIBDIR)\"" \
 	-DEXTENSIONS="$(EXTRA_INIT)" \
 	-DNETWORK -DCURSES_RESET \
